index.d.ts

```diff
@@ -0,0 +1,525 @@
+/**
+ * Access the Electron APIs in both the main and renderer process without having to care about which one you're in.
+ *
+ * @example
+ *
+ * // In the renderer process:
+ * api.app.quit(); // The `app` API is usually only available in the main process.
+ */
+export const api: Electron.AllElectron | Electron.Remote;
+
+/**
+ * Check for various things.
+ */
+export const is: {
+	/**
+	 * Running on macOS.
+	 */
+	macos: boolean,
+
+	/**
+	 * Running on Linux.
+	 */
+	linux: boolean,
+
+	/**
+	 * Running on Windows.
+	 */
+	windows: boolean,
+
+	/**
+	 * Running on the [main process](https://electronjs.org/docs/tutorial/quick-start/#main-process).
+	 */
+	main: boolean,
+
+	/**
+	 * Running on the [renderer process](https://electronjs.org/docs/tutorial/quick-start/#renderer-process).
+	 */
+	renderer: boolean,
+
+	/**
+	 * The app is using [ASAR](https://electronjs.org/docs/tutorial/application-packaging/).
+	 */
+	usingAsar: boolean,
+
+	/**
+	 * Running in development, not in production.
+	 */
+	development: boolean,
+
+	/**
+	 * The app is an Mac App Store build.
+	 */
+	macAppStore: boolean,
+
+	/**
+	 * The app is a Windows Store AppX build.
+	 */
+	windowsStore: boolean
+};
+
+/**
+ * Electron version.
+ *
+ * @example
+ *
+ * '1.7.9'
+ */
+export const electronVersion: string;
+
+/**
+ * Chrome version in Electron.
+ *
+ * @example
+ *
+ * '62.0.3202'
+ */
+export const chromeVersion: string;
+
+type RequireAtLeastOne<T, Keys extends keyof T = keyof T> =
+	Pick<T, Exclude<keyof T, Keys>>
+	& {
+		[K in Keys]-?: Required<Pick<T, K>>
+	}[Keys]
+
+interface _Choices<M, W, L, D> {
+	macos?: M | (() => M);
+	windows?: W | (() => W);
+	linux?: L | (() => L);
+	default?: D | (() => D);
+}
+
+export type Choices<M, W, L, D> = RequireAtLeastOne<_Choices<M, W, L, D>, 'macos' | 'windows' | 'linux'>
+
+/**
+ * Accepts an object with the keys as either `macos`, `windows`, `linux`, or `default`, and picks the appropriate key depending on the current platform.
+ * If no platform key is matched, the `default` key is used if it exists.
+ * If the value is a function, it will be executed, and the returned value will be used.
+ *
+ * @param choices - TODO
+ *
+ * @example
+ *
+ * init({
+ * 	enableUnicorn: util.platform({
+ * 		macos: true,
+ * 		windows: false,
+ * 		linux: () => false
+ * 	})
+ * });
+ */
+export function platform<M = never, W = never, L = never, D = undefined>(choices: Choices<M, W, L, D>): M | W | L | D;
+
+/**
+ * Returns the active window.
+ */
+export function activeWindow(): Electron.BrowserWindow;
+
+/**
+ * Run some JavaScript in the active or given window.
+ *
+ * @param code - TODO
+ * @param window - Default: Current window
+ * @returns A promise for the result of the executed code or a rejected promise if the result is a rejected promise.
+ */
+export function runJS(code: string, window?: Electron.BrowserWindow): Promise<unknown>;
```

Comment:
Use default parameter for `window`.

Reply:
I'm not sure what do you mean. It's marked as an optional parameter, and its default value is documented?

Reply:
Okay, so based on https://github.com/sindresorhus/electron-debug/pull/68#issuecomment-458140940 we're not changing this…